contrib/tracing/README.md

```diff
@@ -234,3 +234,62 @@ Histogram of block connection times in milliseconds (ms).
 [16, 32)               9 |                                                    |
 [32, 64)               4 |                                                    |
 ```
+
+### log_utxocache_flush.py
+
+A BCC Python script to log the cache and index flushes. Based on the
+`utxocache:flush` tracepoint.
+
+```bash
+$ python3 contrib/tracing/log_utxocache_flush.py ./src/bitcoind
+```
+
+```
+Logging utxocache flushes. Ctrl-C to end...
+Duration (µs)   Mode       Coins Count     Memory Usage    Prune    Full Flush
+0               PERIODIC   28484           3929.87 kB      False    False
```

Comment:
We didn't flush here. We should only trace and log on flushes and not each time `FlushStateToDisk()` is called.